extensions-core/druid-bloom-filter/src/main/java/org/apache/druid/query/filter/BloomDimFilter.java

```diff
@@ -56,6 +61,17 @@ public BloomDimFilter(
     this.bloomKFilter = bloomKFilterHolder.getFilter();
     this.hash = bloomKFilterHolder.getFilterHash();
     this.extractionFn = extractionFn;
+    this.filterTuning = filterTuning;
+  }
+
+  @VisibleForTesting
+  public BloomDimFilter(
+      String dimension,
```

Comment:
Unnecessary wrapping, also in many other newly added constructors below in this PR